src/main/scala/org/zalando/spearheads/innkeeper/services/EskipRouteService.scala

```diff
@@ -72,11 +72,22 @@ class EskipRouteService @Inject() (routesRepo: RoutesRepo, routeToEskipTransform
 
     Source.fromPublisher(streamOfRows.mapResult {
       case (routeRow, pathRow) =>
+        val (routeChangeType, timestamp) = if (routeRow.deletedAt.isDefined) {
```

Comment:
I would replace this with pattern matching. The code looks more like Java code than Scala code to me.

```scala
routeRow.deletedAt match {
  case Some(deletedAt) =>
    RouteChangeType.Delete -> deletedAt
  case _ =>
    if (pathRow.updatedAt ...)
```